camus-sweeper/src/main/java/com/linkedin/camus/sweeper/CamusHourlySweeper.java

```diff
@@ -226,21 +256,69 @@ public OutlierCollectorRunner(Properties props, FileSystem fs) {
     public Void call() throws IOException {
       String inputPaths = this.props.getProperty(CamusHourlySweeper.INPUT_PATHS);
       String outputPathStr = this.props.getProperty(CamusHourlySweeper.DEST_PATH);
-      long destinationModTime = fs.getFileStatus(new Path(outputPathStr)).getModificationTime();
+      long destinationModTime = getDestinationModTime(this.fs, outputPathStr, true);
```

Comment:
Actually this is no longer needed since `createStateFileInFolder` in line 281 will first delete existing timestamp files. I've removed it.